openprescribing/measure_definitions/carbon_salbutamol.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Average carbon footprint per salbutamol inhaler",
+  "title": [
+    "Average carbon footprint per salbutamol inhaler"
+  ],
+  "description": [
+    "Mean average carbon impact (kg CO<sub>2</sub>e) per salbutamol inhaler prescribed"
+  ],
+  "numerator_short": "Carbon impact (kg CO<sub>2</sub>e)",
+  "denominator_short": "Salbutamol inhalers",
+  "why_it_matters": [
+    "The NHS has <a href='https://www.longtermplan.nhs.uk/wp-content/uploads/2019/08/nhs-long-term-plan-version-1.2.pdf'> ",
+    "committed to reducing its carbon footprint by 51% by 2025</a> to meet the target in the Climate Change Act, including a shift to lower carbon inhalers",
+    "to deliver a reduction of 4%.</p>",
+    "<p>Salbutamol metered dose inhalers (MDIs) are the single biggest source of carbon emissions from NHS medicines prescribing.",
+    "The 2022-2023 PCN Investment and Impact Fund (IIF) therefore ",
+    "<a href='https://www.england.nhs.uk/publication/network-contract-directed-enhanced-service-investment-and-impact-fund-2022-23-updated-guidance/'>",
+    "includes an indicator</a> focusing on mean carbon emissions per salbutamol inhaler prescribed (kg CO<sub>2</sub>e).",
+    "The aim is to encourage a change in prescribing of salbutamol devices to those associated with lower carbon emissions, resulting in a significant reduction in NHS carbon emissions.</p>",
+    "<p>Data on carbon emissions per salbutamol inhaler can be viewed in the ",
+    "<a href='https://docs.google.com/spreadsheets/d/1eyk2iqNM_4sobDzevZWWcRR00O2tmJO5HwdFZLjhfa4'>",
+    "table here</a>.</p>"
+  ],
+  "tags": [
+    "core",
+    "respiratory"
```

Comment:
for a later date but we should consider reinstalling  https://www.bennett.ox.ac.uk/blog/2020/02/nhs-england-investment-and-impact-fund-a-new-dashboard-on-openprescribing/